lib/evmone/advanced_analysis.cpp

```diff
@@ -92,12 +84,7 @@ AdvancedCodeAnalysis analyze(evmc_revision rev, bytes_view code, const AdvancedE
         block.stack_req = std::max(block.stack_req, opcode_info.stack_req - block.stack_change);
         block.stack_change += opcode_info.stack_change;
         block.stack_max_growth = std::max(block.stack_max_growth, block.stack_change);
-
-        if(state.eos_evm_version > 0 && (opcode == OP_CREATE || opcode == OP_CREATE2)) {
-            block.gas_cost += static_cast<int64_t>(state.gas_params.G_txcreate);
-        } else {
-            block.gas_cost += opcode_info.gas_cost;
-        }
+        block.gas_cost += opcode_info.gas_cost;
```

Comment:
Does this part of code mean we fixed the gas_cost so we do not need to special case OP_CREATE and OP_CREATE2 ?

Reply:
We reverted this part of the code analysis to how it was originally done in evmone.
What we are doing instead is setting the cost of CREATE and CREATE2 to 0 in the original static tables of evmone and we are making the cost dynamic in the create (create_impl) instruction implementation.

Reply:
https://github.com/eosnetworkfoundation/evmone/pull/12/files/12e07e659b8f93dda13516ba0cf790e2537eedec#diff-8dcb083e9ac17ef66c94c342ae2fa8efc85ecb89e9794381780c333805443f57R157-R165